src/data_enrichment/translator.py

```diff
@@ -0,0 +1,47 @@
+from transformers import MarianMTModel, MarianTokenizer
+
+class Translator:
+    def __init__(self):
+        self.models = {
+            'et-en': ('Helsinki-NLP/opus-mt-et-en', 'Helsinki-NLP/opus-mt-en-et'),
+            'ru-en': ('Helsinki-NLP/opus-mt-ru-en', 'Helsinki-NLP/opus-mt-en-ru'),
+            'pl-en': ('Helsinki-NLP/opus-mt-pl-en', 'Helsinki-NLP/opus-mt-en-pl'),
```

Comment:
This entire dictionary should be in a json file , read it from there @pamodaDilranga

Reply:
Created config files and updated the code